.gitignore

```diff
@@ -34,3 +34,4 @@ coverage-report
 packaged.template.yaml
 .cz_py_dependencies_installed
 .aws-sam
+app.zip
```

Comment:
You'll see below ... b/c we now need to deploy the Lambda zip files to _every_ region; we don't use `sam package/deploy` directly.